order.go

```diff
@@ -285,6 +286,24 @@ type RefundLineItem struct {
 	TotalTax   *decimal.Decimal `json:"total_tax,omitempty"`
 }
 
+type PaymentTerms struct {
+	Amount           *decimal.Decimal  `json:"amount,omitempty"`
+	Currency         string            `json:"currency,omitempty"`
+	PaymentTermsName string            `json:"payment_terms_name,omitempty"`
+	PaymentTermsType string            `json:"payment_terms_type,omitempty"`
+	DueInDays        string            `json:"due_in_days,omitempty"`
+	PaymentSchedules []PaymentSchedule `json:"payment_schedules,omitempty"`
+}
+
+type PaymentSchedule struct {
+	Amount                *decimal.Decimal `json:"amount,omitempty"`
+	Currency              string           `json:"currency,omitempty"`
+	IssuedAt              string           `json:"issued_at,omitempty"`
+	DueAt                 string           `json:"due_at,omitempty"`
+	CompletedAt           string           `json:"completed_at,omitempty"`
```

Comment:
Is this a date? If yes, do you need a different type? Also, does this need to be `*` as it can be null?